apps/condo/domains/banking/access/BankAccount.js

```diff
@@ -1,12 +1,20 @@
 /**
  * Generated by `createschema banking.BankAccount 'organization:Relationship:Organization:CASCADE; tin:Text; country:Text; routingNumber:Text; number:Text; currency:Text; approvedAt?:DateTimeUtc; approvedBy?:Text; importId?:Text; territoryCode?:Text; bankName?:Text; meta?:Json; tinMeta?:Json; routingNumberMeta?:Json'`
  */
+const { get } = require('lodash')
+
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
 
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/utils/accessSchema')
 const {
     queryOrganizationEmployeeFor,
     queryOrganizationEmployeeFromRelatedOrganizationFor,
 } = require('@condo/domains/organization/utils/accessSchema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+const { BANK_INTEGRATION_IDS } = require('../constants')
+
 
 /**
```

Comment:
This comment does not provides additional information, that is missing in code.
Also it's may be out of sync with code in the future.
It's easy to figure out abilities just by reading code below.

Reply:
Agree. I did not touch this comment because I did not write it)
Here I just added a permission check to the existing ones